include/tee/optee_ta_avb.h

```diff
@@ -0,0 +1,48 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/* Copyright (c) 2018, Linaro Limited */
+
+#ifndef __TA_AVB_H
```

Comment:
as it related to AVB only, maybe it's better to put all these definitions to `include/avb_verify.h`

Reply:
That's bound to make it harder to keep the information up to date.